UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -213,6 +213,67 @@ public void AffectDamage(float damage, int damageType)
 			UpdateSeverity();
 		}
 
+		public void ExposePressureTemperature(float EnvironmentalPressure, float EnvironmentalTemperature)
```

Comment:
```suggestion
		public void ExposePressureTemperature(float environmentalPressure, float environmentalTemperature)
```

Reply:
this method should be splitted since pressure and temperature are doing two things that don't depend on each other.

Reply:
when would you exposed to temperature and not pressure?

Reply:
You can call the two methods together. It is the logic in the method body what has no cohesion.

Reply:
ok